presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -895,6 +895,25 @@ public void testStructSchemaEvolution()
         assertQuery("SELECT count(*) FROM test_struct_add_column where x.c = 1", "SELECT 0");
     }
 
+    @Test
+    public void testStructFieldsIgnoreCase()
```

Comment:
@bhhari Thanks for adding a test. A few nits:

- name the table test_xxx for consistency, e.g. test_struct_with_uppercase_fields
- add .dwrf extension to the file name for clarify, e.g. struct_with_uppercase_fields.dwrf
- add space after // in comments for readability, e.g. // path to vs. //path to
- fileWrittenBySpark -> sparkFile and oldFilePath -> prestoFile for clarity
- move sparkFile variable closer to where it is used
- run the same set of queries with presto and spark files
- Files.copy statement can be shortened to `Files.copy(sparkFile, prestoPath);`
- delete the test table when done

```
    @Test
    public void testUpperCaseStructFields()
            throws Exception
    {
        assertUpdate("CREATE TABLE test_struct_with_uppercase_field(field0) WITH (FORMAT = 'DWRF') AS " +
                "SELECT CAST((1, 1) AS ROW(SUBFIELDCAP BIGINT, subfieldsmall BIGINT))", 1);

        try {
            assertQuery("SELECT * FROM test_struct_with_uppercase_field", "SELECT (CAST(1 AS BIGINT), CAST(1 AS BIGINT))");
            assertQuery("SELECT field0.SUBFIELDCAP FROM test_struct_with_uppercase_field", "SELECT CAST(1 AS BIGINT)");
            assertQuery("SELECT field0.subfieldcap FROM test_struct_with_uppercase_field", "SELECT CAST(1 AS BIGINT)");

            // delete the file written by Presto and corresponding crc file
            Path prestoPath = getOnlyPath("test_struct_with_uppercase_field");
            Files.delete(prestoPath);
            Files.deleteIfExists(prestoPath.getParent().resolve("." + prestoPath.getFileName() + ".crc"));

            // copy the file written by Spark
            Path sparkFile = Paths.get(this.getClass().getClassLoader().getResource("struct_with_uppercase_field.dwrf").toURI());
            Files.copy(sparkFile, prestoPath);

            assertQuery("SELECT * FROM test_struct_with_uppercase_field", "SELECT (CAST(1 AS BIGINT), CAST(1 AS BIGINT))");
            assertQuery("SELECT field0.SUBFIELDCAP FROM test_struct_with_uppercase_field", "SELECT CAST(1 AS BIGINT)");
            assertQuery("SELECT field0.subfieldcap FROM test_struct_with_uppercase_field", "SELECT CAST(1 AS BIGINT)");
        } finally {
            assertUpdate("DROP TABLE test_struct_with_uppercase_field");
        }
    }

```